### PR TITLE
Feature/sd cli sth tests

### PR DESCRIPTION
**Pull request by a-tylenda**
Asana task: https://app.asana.com/0/1201056252132458/1200969053491981
to verify:

`NO_HOST=1 SCRAMJET_TEST_LOG=1 yarn test:bdd --name="E2E-010" --tags=@starts-host`

4 new test cases were added to CLI feature (from E2E-010 TC-018 to E2E-010 TC-021)

image

### CODE REVIEW

File: bdd/step-definitions/e2e/host-steps.ts [hunk 1]
```diff
@@ -28,10 +28,15 @@ let actualHealthResponse: any;
 let actualStatusResponse: any;
 let actualApiResponse: Response;
 let sequence: SequenceClient;
+let sequence1: SequenceClient;
```
Comment (MichalCz) [inline bdd/step-definitions/e2e/host-steps.ts:31]: @a-tylenda could you perhaps refactor this a bit. These could be pushed to separate files as this one is growing quite a lot.

**Review by MichalCz — COMMENTED** [review]
Ok, this looks good. Only let's move some of those test implementations to separate files.

File: bdd/step-definitions/e2e/host-steps.ts (hunk 1 above)
Comment (a-tylenda) [reply bdd/step-definitions/e2e/host-steps.ts:31]: fixed in https://github.com/scramjetorg/transform-hub/pull/82

**Review by MichalCz — APPROVED** [review]
LGTM. :)

**Comment by MichalCz** [comment]
Checking verify

**Comment by MichalCz** [comment]
Verify failed - host is not running.

**Comment by MichalCz** [comment]
obraz

:(

**Review by MichalCz — CHANGES_REQUESTED** [review]
Not working... :(

**Comment by a-tylenda** [comment]
my bad! Sorry! please try again witj this command:
`NO_HOST=1 DEVELOPMENT=true SCRAMJET_TEST_LOG=true yarn test:bdd --name="E2E-010" --tags=@starts-host`

**Review by patuwwy — APPROVED** [review]

**Review by MichalCz — APPROVED** [review]
obraz

**Pull request merged**